index.js

```diff
@@ -6,8 +6,12 @@ const rootURI = 'https://www.googleapis.com';
 export const refreshTokenURI = 'https://www.googleapis.com/oauth2/v4/token';
 const uploadExistingURI = id =>
     `${rootURI}/upload/chromewebstore/v1.1/items/${id}`;
-const publishURI = (id, target) =>
-    `${rootURI}/chromewebstore/v1.1/items/${id}/publish?publishTarget=${target}`;
+
+const publishURI = (id, target = 'default', deployPercentage) => {
```

Comment:
See notes in #90 regarding the preferred style (i.e. the object)

Reply:
Thanks, I will update my PR with those recommendations.

Reply:
I've also updated the method to use the URL object as recommended in #90.